inferno-core/src/Inferno/Module/Prelude.hs

```diff
@@ -650,6 +660,18 @@ module Base
       | None -> default
     };
 
+  // TODO fix parser and re-introduce this
```

Comment:
(minor) we need this on the PR?

Reply:
It's not super easy to do, so I'll do it in a future PR